docs/hub/models-inference.md

```diff
@@ -1,30 +1,76 @@
 # Inference Providers
 
-Please refer to the [Inference Providers Documentation](https://huggingface.co/docs/inference-providers) for detailed information.
+Hugging Face's model pages have free inference for thousands of models, so you can try them all out right in the browser. It's also powered by Inference Providers.
 
-## What is HF-Inference API?
+Inference Providers give developers streamlined, unified access to hundreds of machine learning models, powered by the best serverless inference partners. 👉 **For complete documentation, visit the [Inference Providers Documentation](https://huggingface.co/docs/inference-providers)**.
 
-HF-Inference API is one of the many providers available on the Hugging Face Hub.
-It is deployed by Hugging Face ourselves, using text-generation-inference for LLMs for instance. This service used to be called “Inference API (serverless)” prior to Inference Providers. 
+## Inference Providers on the Hub
 
-For more details about the HF-Inference API, check out its [dedicated page](https://huggingface.co/docs/inference-providers/providers/hf-inference).
+Inference Providers is deeply integrated with the Hugging Face Hub, and you can use it in a few different ways:
 
-## What technology do you use to power the HF-Inference API?
+- **Interactive Widgets** - Test models directly on model pages with interactive widgets that use Inference Providers under the hood. Check out the [DeepSeek-R1-0528 model page](https://huggingface.co/deepseek-ai/DeepSeek-R1-0528) for an example.
+- **Inference Playground** - Easily test and compare chat completion models with your prompts. Check out the [Inference Playground](https://huggingface.co/playground) to get started.
+- **Search** - Filter models by inference provider on the [models page](https://huggingface.co/models?inference_provider=all) to find models available through specific providers.
 
-The HF-Inference API is powered by [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index) under the hood.
+## Build with Inference Providers
 
-## Why don't I see an inference widget, or why can't I use the API?
+You can integrate Inference Providers into your own applications using our SDKs or HTTP clients. Here's a quick start with Python and JavaScript, for more details, check out the [Inference Providers Documentation](https://huggingface.co/docs/inference-providers).
 
-For some tasks, there might not be support by any Inference Provider, and hence, there is no widget.
+<hfoptions id="inference-providers-quick-start">
 
-## How can I see my usage?
+<hfoption id="python">
 
-To check usage across all providers, check out your [billing page](https://huggingface.co/settings/billing).
+```python
+from huggingface_hub import InferenceClient
```

Comment:
make these complete w/ huggingface_hub login

Reply:
I would keep "as is" to be honest. The snippet already includes `api_key=os.environ["HF_TOKEN"],` (which by the way is not even needed but good to let the users know about the env variable). This is the code snippet we generate on model pages.

Reply:
(I just realized we should go for `os.getenv("HF_TOKEN")` instead of `os.environ["HF_TOKEN"]`... First one will work if token is saved in file instead of env variable while the second will fail even though the user is logged in. I'll make a pass to fix that everywhere :disappointed: